PurchasesCoreSwift/Misc/ISOPeriodFormatter.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ISOPeriodFormatter.swift
+//  PurchasesCoreSwift
+//
+//  Created by Joshua Liebowitz on 6/30/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+import StoreKit
+
+@available(iOS 11.2, macOS 10.13.2, tvOS 11.2, *)
+
+// TODO(post-migration): Switch public back to <> (nothing)
+// TODO(post-migration): remove @objc since this is internal only
+@objc(RCISOPeriodFormatter) public class ISOPeriodFormatter: NSObject {
+
+    @objc public func stringFromProductSubscriptionPeriod(_ period: SKProductSubscriptionPeriod) -> String {
```

Comment:
I think in Swift this should be named
```swift
string(from productSubscriptionPeriod:)
```
or 
```swift
string(fromProductSubscriptionPeriod period:)
```
which would translate to the same objc

Reply:
Switching to `string(fromProductSubscriptionPeriod period: SKProductSubscriptionPeriod)`
`string(from productSubscriptionPeriod:)` doesn't work for objc 😢

Reply:
that makes sense - the argument label is what actually gets translated into objective c, and the parameter name is skipped